website/docs/reference/resource-configs/schema.md

```diff
@@ -5,10 +5,66 @@ description: "Schema - Read this in-depth guide to learn about configurations in
 datatype: string
```

Comment:
description: "Override the default schema when dbt creates resources in your data platform."